cmd/encryption-methods.go

```diff
@@ -0,0 +1,269 @@
+package cmd
```

Comment:
```suggestion
package cmd
```

missing copyright header